ProjectOne/sass/pages/_login.scss

```
@@ -0,0 +1,19 @@
+.login {
+  &__form .form__legend {
+    margin: 0;
+  }
+
+  &__form .form__para {
+    margin-bottom: 4rem;
+  }
+
+  &__form .form__cta {

```

đoạn này có chung &__form bọc ở ngoài sao k đưa ra luôn
```suggestion
  &__form .form__legend {
    margin: 0;
  }

  &__form .form__para {
    margin-bottom: 4rem;
  }

  &__form {
    .form__cta {}
    .form__cta {}
    ...
  }
```

---

Vâng a. em sẽ fix lại